scripts/db-test-utils.py

```
@@ -0,0 +1,272 @@
+                    logger.info(f"   {i+1}. {row['email']} (超级用户: {row['is_superuser']}, 激活: {row['is_active']})")
+                elif query_info['name'] == '标签':
+                    logger.info(f"   {i+1}. {row['name']} - {row['description']}")
+                elif query_info['name'] == '提示词标题':
+                    logger.info(f"   {i+1}. {row['name']} - {row.get('description', 'N/A')}")
+                elif query_info['name'] == '内容项目':
+                    logger.info(f"   {i+1}. {row['title']} ({row['type']})")
+                    
+        except Exception as e:
+            logger.error(f"❌ 获取 {query_info['name']} 信息失败: {e}")
+
+def main():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)